helm/cluster-api/files/controlplane/patches/versions/v1alpha3/kubeadmcontrolplanes.controlplane.cluster.x-k8s.io.yaml

```diff
@@ -0,0 +1,774 @@
+- op: replace
```

Comment:
Kustomize patch with v1alpha3 OPEN API schema for `kubeadmcontrolplanes.controlplane.cluster.x-k8s.io`.